src/contracts/env/FlowSwapPair.cdc

```diff
@@ -22,36 +22,36 @@ pub contract FlowSwapPair: FungibleToken {
   access(contract) let token2Vault: @TeleportedTetherToken.Vault
 
   // Defines token vault storage path
-  pub let TokenStoragePath: StoragePath
+  access(all) let TokenStoragePath: StoragePath
 
   // Defines token vault public balance path
-  pub let TokenPublicBalancePath: PublicPath
+  access(all) let TokenPublicBalancePath: PublicPath
 
   // Defines token vault public receiver path
-  pub let TokenPublicReceiverPath: PublicPath
+  access(all) let TokenPublicReceiverPath: PublicPath
 
   // Event that is emitted when the contract is created
-  pub event TokensInitialized(initialSupply: UFix64)
+  access(all) event TokensInitialized(initialSupply: UFix64)
 
   // Event that is emitted when tokens are withdrawn from a Vault
-  pub event TokensWithdrawn(amount: UFix64, from: Address?)
+  access(all) event TokensWithdrawn(amount: UFix64, from: Address?)
 
   // Event that is emitted when tokens are deposited to a Vault
-  pub event TokensDeposited(amount: UFix64, to: Address?)
+  access(all) event TokensDeposited(amount: UFix64, to: Address?)
```

Comment:
Same comment about removing these

Reply:
These comments apply to every swap pair you have, good to make sure they are all equivalent

Reply:
Done